pkg/testutil/testutil_test.go

```diff
@@ -84,17 +84,18 @@ func (s *TestUtilSuite) TestDeployment(c *C) {
 	}
 
 	cm := NewTestConfigMap()
-	cm, err = cli.CoreV1().ConfigMaps(ns.GetName()).Create(context.TODO(), cm, metav1.CreateOptions{})
+	cm, err = cli.CoreV1().ConfigMaps(ns.GetName()).Create(ctx, cm, metav1.CreateOptions{})
 	c.Assert(err, IsNil)
 	err = cli.CoreV1().ConfigMaps(ns.GetName()).Delete(context.TODO(), cm.GetName(), metav1.DeleteOptions{})
 	c.Assert(err, IsNil)
 }
 
 func testCRs(c *C, cli crclientv1alpha1.CrV1alpha1Interface, namespace, poKind, poName string) {
```

Comment:
Pass it in from above
```suggestion
func testCRs(c *C, ctx context.Context, cli crclientv1alpha1.CrV1alpha1Interface, namespace, poKind, poName string) {
```